src/common/components/nav/nav.component.js

```diff
@@ -37,6 +45,34 @@ class NavController{
     this.getNav().subscribe((structure) => {
       this.menuStructure = structure;
     });
+
+    this.subscription = this.sessionService.sessionSubject.subscribe( () => this.sessionChanged() );
+
+    this.templateUrl = this.$window.screen && this.$window.screen.width < 991 ? mobileTemplate.name : desktopTemplate.name;
```

Comment:
This doesn't seem to be working. It is always 1920 for me no matter what my chrome window size is or where I move the inspector. Also can this be done in CSS? I feel like you could put both on the template and use http://getbootstrap.com/css/#responsive-utilities-classes to show/hide them. That would also allow the page to switch menus responsively as the user resizes the page.